descholar-ceo/find-flight-info
# User Login

#### What does this PR ​ do​ ?
It allows user to login

#### Description of Task to be completed?
* Create Login component
* Create a handleLoginTyping in handlers
* Add link to login page on signup page
* Create a routing to login page in App.js file

#### How should this be manually tested?
* Clone this repository by `git clone https://github.com/descholar-ceo/find-flight-info`
* Type these commands : 
    *  `cd find-flight-info` to go in the root directory of the project
    * `npm install` or `yarn add` to install all the packages
    * `npm start` or `yarn start`
* Then go in the browser, you will see a default opened page is signup
* Signup and and after signing up, 
* Enter this url in url line `http://localhost:3000/#/login`

#### Any background context you want to provide?
* You have to install [nodejs](https://nodejs.org/en/) version 6 or higher prior to test this application

#### What are the relevant pivotal tracker stories?

#### Screenshots (​ if​ appropriate)
![02 loginPage](https://user-images.githubusercontent.com/56886064/74945639-c4fefd80-5400-11ea-9732-f9bca620bbb1.png)

#### Questions:
No Question
